### DISCUSSION TITLE
Download all clips of a detect event?

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#15405] [Ask A Question] [answered]
I am curious if I can export in bulk a bunch of detected events. I have bark events that happen multiple times a day and night and I want them to automatically be exported. Is this possible? For example if I search for a camera history by events filtered by bark I get a bunch of thumbnails for when these events were triggered. I dont think they're time boxed "clips"? Anyway, would be great if I could have these automatically exported somehow or bulk.

**Comment by NickM-27** [comment]
You can use a script or automation to do this

**Reply by moogoos** [reply]
I assume using the API? Any little bit of guidance to help me get started?

**Reply by NickM-27** [reply, marked as answer]
Yes, using the export api or the export service in home assistant
